packages/near-fast-auth-signer-e2e-tests/tests/sign/sign-transaction.spec.ts

```diff
@@ -116,6 +116,7 @@ describe('Sign transaction', () => {
       networkId: 'testnet',
       provider:  { type: 'JsonRpcProvider', args: { url: 'https://rpc.testnet.near.org' } },
       signer:    { type: 'InMemorySigner', keyStore: new InMemoryKeyStore() },
+      // @ts-ignore
```

Comment:
Plase explain the @ts-ignore

Reply:
Removed.
It stops my tests from running as there's unimplemented with error `Property useLocalViewExecution is missing in type`, had to disable the warning before running mine, guess it's not so for every other person.